packages/massa-web3/src/experimental/publicAPI.ts

```diff
@@ -193,12 +221,28 @@ export class PublicAPI {
     return this.connector.get_status()
   }
 
-  async sendOperation(data: OperationInput): Promise<OperationId> {
+  async fetchChainId(): Promise<number> {
+    return this.getStatus().then((r) => r.chain_id)
+  }
```

Comment:
Its a bit sad to make a rpc call each time we need a info from getStatus.
Lets see how it will be used, but imho, we should fetch the constant data of getStatus (like chainId)  once for all at initialization of the client.
But maybe its more an optimization